packages/website/posts/2022-03-29-50M-uploads.mdx

```diff
@@ -0,0 +1,30 @@
+---
+title: 'NFT.Storage crosses 50M uploads'
+description: NFT.Storage has grown to 50M uploads. Here's to the next 500M!
+author: David Choi
+thumbnail: https://user-images.githubusercontent.com/89210666/160478599-0de7cfdd-da01-4df5-a24d-12dcc01db259.png
+date: Mar 29, 2022
+tags:
+  - milestone
+---
+
+We're excited to announce that over the weekend, NFT.Storage crossed 50 million user uploads to our service! It's a big milestone that shows both the overall growth of the NFT space, as well as increased awareness within the space of the importance of handling off-chain NFT data the right way. The NFT ecosystem is increasingly seeing the importance of making references off-chain data [content addressed](https://docs.ipfs.io/concepts/content-addressing/) (using IPFS CIDs so no one can argue what the NFT is for), and ensuring this data is always accessible on IPFS via these content addresses (with NFT.Storage as one perpetual storage option with long-term provable storage via Filecoin, but also utilizing IPFS's flexibility to store wherever else [makes them feel comfortable](https://nft.storage/blog/post/2021-12-14-storage-layer-maximalism/)).
```

Comment:
“the right way” and “so no one can argue what the NFT is for” are both pretty ambiguous.

i’d change “the right way” to “securely” but i’m not entirely sure what is being conveyed with “argue what the NFT is for” so i’m not sure what to suggest.

Reply:
"argue what the NFT is for" = immutable link to the content itself (you can prove that your NFT is for this pikachu if you say it is). i'll make the language clearer.